src/resources/2022/AutumnAton.ts

```diff
@@ -169,3 +171,36 @@ export function zoneItems(): 3 | 4 | 5 {
 export function seasonalItems(): 1 | 2 {
   return currentUpgrades().includes("cowcatcher") ? 2 : 1;
 }
+
+type Difficulty = "low" | "mid" | "high";
+const UNIQUES = {
+  outdoor: {
+    low: { index: 4, item: $item`autumn leaf` },
+    mid: { index: 2, item: $item`autumn debris shield` },
+    high: { index: 6, item: $item`autumn leaf pendant` },
+  },
+  indoor: {
+    low: { index: 0, item: $item`AutumnFest ale` },
+    mid: { index: 3, item: $item`autumn-spice donut` },
+    high: { index: 7, item: $item`autumn breeze` },
+  },
+  underground: {
+    low: { index: 1, item: $item`autumn sweater-weather sweater` },
+    mid: { index: 5, item: $item`autumn dollar` },
+    high: { index: 8, item: $item`autumn years wisdom` },
+  },
+};
+
+function fallbotEnv(location: Location): "underground" | "indoor" | "outdoor" {
+  const env = location.environment as EnvironmentType;
+  return env === "underwater" ? "underground" : env;
```

Comment:
After a small test, sending autumnaton with a cowcatcher to the Briny Deeps twice, first time it brought back ONLY an autumnFest ale, second time ONLY autumn years wisdom. 

These are the cowcatcher items, which seems to mean we don't get anything at all for sending it underwater.

It probably won't be relevant since we can't really get anything from underwater locations anyway, but just for accuracy